sefaria/helper/topic.py

```diff
@@ -1105,6 +1105,13 @@ def update_topic(topic, **kwargs):
     if "description" in kwargs or "categoryDescription" in kwargs:
         topic.change_description(kwargs.get("description", None), kwargs.get("categoryDescription", None))
 
+    if "image" in kwargs:
```

Comment:
As we discussed verbally, Might be worth adding a line of inline comment explaining that this is to validate against orphaned captions.